packages/react-form/src/hooks/submit.ts

```diff
@@ -7,18 +7,21 @@ import {
   SubmitResult,
   FieldBag,
   FormError,
+  DynamicListBag,
 } from '../types';
 import {
   propagateErrors,
   validateAll,
   getValues,
   makeCleanFields,
+  makeCleanDynamicList,
 } from '../utilities';
 
 export function useSubmit<T extends FieldBag>(
   onSubmit: SubmitHandler<FormMapping<T, 'value'>> = noopSubmission,
   fieldBag: T,
   makeCleanAfterSubmit = false,
+  dynamicLists?: DynamicListBag,
```

Comment:
Could we call it `dinamicListBag` for the sake of consistency?

Reply:
I was thinking of following the same naming as useForm since it was called `dynamicLists` there but I think since it uses fieldBag naming here, it's fair to change it too to be consistent. Thanks!